lib/scala/pkg/src/main/resources/default/src/Main.enso

```diff
@@ -2,6 +2,7 @@ from Standard.Base import all
 from Standard.Table import all
 from Standard.Database import all
 from Standard.AWS import all
+import Standard.Visualization
```

Comment:
Can we create a plan for this _user unfriendly_ `import` to be removed again, @4e6 & @hubertp? Can we turn the steps of such plan into issue tracker entries?